src/main/java/com/redhat/cloud/policies/engine/db/repositories/PoliciesRepository.java

```diff
@@ -44,22 +48,36 @@ public List<Policy> getEnabledPolicies(String accountId) {
     @CacheResult(cacheName = ACCOUNT_LATEST_UPDATE_CACHE_NAME)
     LocalDateTime findLatestUpdate(String accountId) {
         LOGGER.debugf("Finding latest policies update time for account %s", accountId);
-        String hql = "SELECT latest FROM AccountLatestUpdate WHERE accountId = :accountId";
         try {
-            return statelessSessionFactory.getCurrentSession().createQuery(hql, LocalDateTime.class)
-                    .setParameter("accountId", accountId)
-                    .getSingleResult();
+            if (orgIdConfig.isUseOrgId()) {
+                String hql = "SELECT latest FROM OrgIdLatestUpdate WHERE orgId = :orgId";
+                return statelessSessionFactory.getCurrentSession().createQuery(hql, LocalDateTime.class)
+                        .setParameter("orgId", accountId)
+                        .getSingleResult();
+            } else {
+                String hql = "SELECT latest FROM AccountLatestUpdate WHERE accountId = :accountId";
+                return statelessSessionFactory.getCurrentSession().createQuery(hql, LocalDateTime.class)
+                        .setParameter("accountId", accountId)
+                        .getSingleResult();
+            }
         } catch (NoResultException e) {
-            LOGGER.debugf("No latest policies update time found, using default value", accountId);
+            LOGGER.debug("No latest policies update time found, using default value");
             return LocalDateTime.MIN;
         }
     }
 
     private List<Policy> findEnabledPoliciesByAccount(String accountId) {
-        String hql = "FROM Policy WHERE accountId = :accountId AND enabled IS TRUE";
-        return statelessSessionFactory.getCurrentSession().createQuery(hql, Policy.class)
-            .setParameter("accountId", accountId)
-            .getResultList();
+        if (orgIdConfig.isUseOrgId()) {
+            String hql = "FROM Policy WHERE orgId = :orgId AND enabled IS TRUE";
+            return statelessSessionFactory.getCurrentSession().createQuery(hql, Policy.class)
+                    .setParameter("orgId", accountId)
```

Comment:
Same here, the value should be `orgId`. Same solution suggestion: duplicate the method and check `orgIdConfig.isUseOrgId()` to determine which method should be called.